resource-timing/resource_initiator_types.html

```diff
@@ -135,6 +135,8 @@
     addEntryIfExists(entries, expected_entries, pathname + 'empty.py?favicon', 'link');
     addEntryIfExists(entries, expected_entries, pathname + 'eventsource.py?id=eventsource', 'other');
 
+    assert_greater_than(entries.length, 0, 'There should be resrouce entries.');
```

Comment:
I'm considering whether this assert should be here or inside the test_resource_entries function instead.

Any thoughts?

Reply:
Based on discussion with WebPerfWG - we should look into modifying the test to use performance observer to wait for all resources to load.